csvcubed/csvcubed/readers/preconfiguredtemplates.py

```diff
@@ -0,0 +1,81 @@
+"""
+Pre-configured JSON Templates
+-----------------------------
+
+Functionality to help augment JSON files with configuration from some pre-configured templates.
+"""
+import logging
+
+from typing import Dict, Any, Optional
+from requests.exceptions import JSONDecodeError
+
+from csvcubed.utils.cache import session
+
+def _get_template_file_from_template_lookup(template_value: str, version_module_path: str) -> str:
+    """
+    Given the `from_template` value, look up the template in the git repo
+    """
+
+    template_lookup_url =  f"https://raw.githubusercontent.com/GSS-Cogs/csvcubed/main/csvcubed/csvcubed/readers/{version_module_path}/templates/preset_column_config.json"
+    template_lookup_response = session.get(template_lookup_url)
+    logging.debug("The template lookup/index file: %s", template_lookup_url)
+   
+    if template_lookup_response.status_code != 200:
+        logging.warning("Status code: %s.", template_lookup_response.status_code)
+        logging.debug("Response: %s.", template_lookup_response)
+
+    try:
+        template_lookup = template_lookup_response.json()
+    except JSONDecodeError as e:
+        raise Exception(f"Could not access template lookup file at {template_lookup_url}") from e
+
+    template_file = template_lookup.get(template_value)
+    if template_file == None:
+        raise Exception("Couldn't find template your looking for.")
+
+    return template_file
+
+
+def _get_propeties_from_template_file(template_file: str, version_module_path: str) -> dict:
```

Comment:
can we apply the comments on `_get_template_file_from_template_lookup` to this function as well please.